pkgs/applications/networking/remote/citrix-workspace/generic.nix

```diff
@@ -29,7 +29,7 @@ let
     paths = [ opencv4 ];
     postBuild = ''
       for so in ${opencv4}/lib/*.so; do
-        ln -s "$so" $out/lib/$(basename "$so").407
+        ln -s "$so" $out/lib/$(basename "$so").410
```

Comment:
Won't this fail with earlier versions that use `407`?

Reply:
I would like to get this in so I don't have to maintain out-of-tree patches in my local tree for building my local config. I went ahead and did a `|| true` for both 407 and 410 for now, which builds fine locally. You're right that this package should be rewritten with just the latest version in mind... it's a terrible idea and a huge burden to do a take-your-pick for package versions. It doesn't even seem like Citrix has links to the older versions available for download (https://www.citrix.com/downloads/workspace-app/linux/) ... I will take up rewriting it some weekend in the near future :)